docs/guides/core-concepts/introduction-to-cypress.mdx

```diff
@@ -1197,15 +1180,14 @@ cy.get('#modal').should('not.exist')
 
 <strong>Core Concept</strong>
 
-By adding [`.should('not.exist')`](/api/commands/should) to any DOM command,
-Cypress will reverse its default assertion and automatically wait until the
-element does not exist.
+If you add [`.should('not.exist')`](/api/commands/should) to any DOM command,
+Cypress will skip its default existence assertion.
```

Comment:
I had to re-read this to understand it. Maybe it could be made more clear?

```suggestion
If you add [`.should('not.exist')`](/api/commands/should) to any DOM command,
Cypress will skip the default assertion, which is to check existence.
```

Not even sure that's grammatically correct... I asked ChatGPT:

![image](https://user-images.githubusercontent.com/1151760/220998750-db4e64d3-a08f-4501-86a3-a1c3de43bb52.png)

I kinda like the last one:

```suggestion
If you want to disable the default existence assertion for an element in Cypress, you can add [`.should('not.exist')`](/api/commands/should) to any DOM command.
```

Reply:
Went with
> If you want to disable the default existence assertion, you can add [`.should('not.exist')`](/api/commands/should) to any DOM command.